### PR TITLE
[MM-19856] Add back button when navigating outside of the mattermost team within the app

### PR DESCRIPTION
**Pull request by Willyfrog**
Before submitting, please confirm you've
 - [x] executed `npm run lint:js` for proper code formatting

Please provide the following information:

**Summary**
Provide an extra bar with a `back` option to go back in history. This bar only appears when the app navigates outside of a mattermost team.

**Issue link**

[MM-19856](https://mattermost.atlassian.net/browse/MM-19856)

**Test Cases**

1. logout from the community-* server
2. login using onelogin
expected:
  - extra bar appears

3. clicking on back 
expected:
  - takes you to the previous screen (login in this case)  and it disappears

3. completing the login process
expected: 
  - takes you to the team and the bar disappears

** Additional notes **
- Added awesome font to render the back symbol
- fixed a but on `isTeamURL`, we weren't checking the origin on it relying on previous checks to work.

### CODE REVIEW

**Comment by Willyfrog** [comment]
Forgot the screenshots
Screenshot 2020-01-31 at 12 32 39
Screenshot 2020-01-31 at 12 32 55

File: src/browser/components/ExtraBar.jsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        <div
+          className={'container-fluid'}
+          onClick={this.handleBack}
+        >
+          <span
+            id={'back_button_icon'}
+            className={'backIcon fa fa-1x fa-angle-left'}
+            title={'Back Icon'}
+          />
+          <span className={'backLabel'}>{'Back'}</span>
```
Comment (deanwhillier) [inline src/browser/components/ExtraBar.jsx]: We should use a more semantic `<button>` or tag for this and make sure keyboard navigation will work. See the `link` button example at the top of the React Bootstrap page: https://react-bootstrap.github.io/components/buttons/.

**Comment by Willyfrog** [comment]
For some reason, I thought I should have a dark/clear version. Sorry about that.

I've matched the one from `Password reset`, bar is now about 36, added button.

I left the logic for clear/dark version in just in case we want to use that in the future, it just doesn't do anything with css.
Screenshot 2020-02-06 at 10 55 28
Screenshot 2020-02-06 at 10 55 48

**Comment by Willyfrog** [comment]
I did try the open sans family, but it looked way worse, like bolded even though it was normal, I'll review that again just in case I can make it to work.

**Comment by andrewbrown00** [comment]
@Willyfrog or @deanwhillier will you post screenshots or a link to the build I can download and test, thanks guys!

**Comment by Willyfrog** [comment]
link to artifacts on the build: https://app.circleci.com/jobs/github/mattermost/desktop/5481/artifacts,

I'll DM  a video, I can't post it here

**Comment by Willyfrog** [comment]
Force pushed as there was a lot of files from master being re-added

**Pull request merged**